Validation/RecoTrack/python/HLTmultiTrackValidator_cfi.py

```diff
@@ -9,14 +9,14 @@
 hltMultiTrackValidator.label   = cms.VInputTag(cms.InputTag("hltPixelTracks"))
 hltMultiTrackValidator.beamSpot = cms.InputTag("hltOnlineBeamSpot")
 hltMultiTrackValidator.ptMinTP  = cms.double( 0.4)
-hltMultiTrackValidator.lipTP    = cms.double(35.0)
-hltMultiTrackValidator.tipTP    = cms.double(70.0)
-hltMultiTrackValidator.histoProducerAlgoBlock.generalTpSelector.ptMin = cms.double( 0.4)
-hltMultiTrackValidator.histoProducerAlgoBlock.generalTpSelector.lip   = cms.double(35.0)
-hltMultiTrackValidator.histoProducerAlgoBlock.generalTpSelector.tip   = cms.double(70.0)
+hltMultiTrackValidator.lipTP    = cms.double(30.0)
```

Comment:
Please drop type specs as suggested here: 6.17 in http://cms-sw.github.io/cms_coding_rules.html#python  
Since with the change the mis-spelled variable name would give error. 
See for instance[ https://github.com/cms-sw/cmssw/pull/33046/files](https://github.com/cms-sw/cmssw/pull/33046/files)
Thanks

Reply:
Done in commit 2cafb1 for the updated values, as well as for those that were already there.